python/dgl/contrib/dis_kvstore.py

```diff
@@ -220,29 +220,31 @@ def _push_handler(self, name, ID, data):
             a vector storing the ID list.
         data : tensor (mx.ndarray or torch.tensor)
             a tensor with the same row size of id
+        target : target tensor (mx.ndarray or torch.tensor)
+            the target tensor
```

Comment:
what is target tensor?

Reply:
It's self._data_store, not a tensor. The comment line is wrong.